stdlib/asm/math/u64.masm

```diff
@@ -306,10 +306,10 @@ end
 #! [b_hi, b_lo, a_hi, a_lo, ...] -> [c, ...], where c = 1 when a == b, and 0 otherwise.
 export.unchecked_eq
     movup.2
-    u32checked_eq
+    u32assert2 eq
     swap
     movup.2
-    u32checked_eq
+    u32assert2 eq
     and
 end
```

Comment:
I think doing `u32assert2` may not be needed here. Seems like this might have been a "bug".